.ci/scripts/send-pr.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -uexo pipefail
+
+readonly APM_AGENT=${1}
+readonly APM_AGENT_SPECS_DIR=${2}
+readonly APM_AGENT_REPO_NAME="apm-agent-${APM_AGENT}"
+readonly GIT_DIR=".ci/git"
+readonly APM_AGENT_REPO_DIR="${GIT_DIR}/${APM_AGENT_REPO_NAME}"
+
+git clone "https://github.com/elastic/${APM_AGENT_REPO_NAME}" "${APM_AGENT_REPO_DIR}"
+
+mkdir -p "${APM_AGENT_REPO_DIR}/${APM_AGENT_SPECS_DIR}"
+cp tests/agents/gherkin-specs/*.feature "${APM_AGENT_REPO_DIR}/${APM_AGENT_SPECS_DIR}"
+
+cd ${APM_AGENT_REPO_DIR}
+git checkout -b update-feature-files-$(date "+%Y%m%d%H%M%S")
+echo "Copying feature files to the ${APM_AGENT_REPO_NAME} repo"
+git status
+git add ${APM_AGENT_SPECS_DIR}
+git commit -m "ci: Synchronized BDD specs"
+
+if [[ "${DO_SEND_PR}" == "true" ]]; then
+    hub pull-request \ 
+        -p \                                  # push the branch to the remote
+        -b master \                           # target branch 
```

Comment:
Is this required? I mean, if the upstream got a different branch as default one, then this should be changed, so what about using the default behavior?

Reply:
Yeah, makes sense. Ideally it should be master, but we cannot make sure teams are using other branch. Agreed in removing that flag

Reply:
Removed! thanks!